tests/Integration/Service/Server/Generator/AccessTokenResponseGeneratorTest.php

```diff
@@ -138,16 +163,16 @@ public function testGenerateWitInvalidCredentials(): void
             ]
         );
 
-        $result = $this->subject->generate($request, $this->createResponse(), $registration->getIdentifier());
+        $result = $this->subject->generate($request, $this->createResponse(), $keyChain->getIdentifier());
 
         $this->assertInstanceOf(ResponseInterface::class, $result);
         $this->assertEquals(401 , $result->getStatusCode());
```

Comment:
Not related to this PR, but extra space here.

Reply:
thx, fixed.